distributed/bokeh/scheduler.py

```diff
@@ -927,12 +927,18 @@ class WorkerTable(DashboardComponent):
     This is two plots, a text-based table for each host and a thin horizontal
     plot laying out hosts by their current memory use.
     """
+    excluded_names = {'executing', 'in_flight', 'in_memory', 'ready', 'time'}
 
     def __init__(self, scheduler, width=800, **kwargs):
         self.scheduler = scheduler
         self.names = ['worker', 'ncores', 'cpu', 'memory', 'memory_limit',
                       'memory_percent', 'num_fds', 'read_bytes', 'write_bytes',
                       'cpu_fraction']
+        workers = self.scheduler.workers.values()
+        # Keep self.custom_info_names in the same order as one the workers (Python 3.6+ only)
```

Comment:
Ah, good point.  What are your thoughts on ordering?  I personally don't care that much, but for your intended application it may be more important.  You probably have a better sense of what is important here than I do.  I probably just default to promising as little as possible until necessary :)

Reply:
I like the fact that you can control the ordering by how you register metrics. I don't think it is that crucial in practice though, especially since it is in alphabetical order you can always rename your metric if you really care.